### ISSUE TITLE
Textfield autofocus not applied

### ISSUE THREAD

**Issue by HamburgerJungeJr** [material-components/material-components-web#4402]
### What MDC Web Version are you using?

material-components-web@0.42.1

### What browser(s) is this bug affecting?

Latest Firefox and latest Chrome

### What OS are you using?

Ubuntu 18.10

### What are the steps to reproduce the bug?
Create a Textfield and add autofocus-Property
https://codepen.io/hamburgerjungejr/pen/VgxKYN

### What is the expected behavior?

Textfield has focus state and cursor is placed in textfield

### What is the actual behavior?
Cursor is not placed in textfield and textfield is not in focus state

### Any other information you believe would be useful?
There is a closed issue
https://github.com/material-components/material-components-web/issues/1529
But the bug seems to still occure

**Comment by abhiomkar** [comment]
Thanks for reporting this issue. I'm not able to reproduce this when text field rendered in [full page view](https://codepen.io/hamburgerjungejr/full/VgxKYN). Codepen doesn't seem to like `autofocus` attribute when rendered in Edit view.

**Comment by kfranqueiro** [comment]
This is probably a feature in Codepen's case, to avoid the split-pane preview from stealing focus while editing, since it refreshes as you're making changes.

Have you seen this issue outside the context of Codepen?

**Comment by HamburgerJungeJr** [comment]
I just tried it again. And I noticed it seemes to be a problem with Firefox. 
The input is not visible at first (display:none) because of a loading screen and Firefox sets the focus not for invisible inputs. 
Other browser do so because the specification is not so clear about how to implement autofocus. So I have to find a workaround with javascript to focus the input when the loading screen closes.
